### PR TITLE
Add auth suffixes and choose keycloak for test & auth0 for prod

### PR DESCRIPTION
**Pull request by tangrammer**
Relates https://github.com/akvo/akvo-lumen/pull/2390

### CODE REVIEW

**Comment by tangrammer** [comment]
BTW: these values are the currently running ones in k8s thus I edited them directly using Google cloud k8s UI

File: k8s/config/prod.yaml [hunk 1]
```diff
@@ -4,13 +4,15 @@ metadata:
...
   auth0.url: https://akvofoundation.eu.auth0.com
-  auth.public.client.id: 7Ze0JP6Jjf7zItGy3ekyuTvwGyik4Lay
-  auth.url: https://akvofoundation.eu.auth0.com
   email.host: mail.akvo.org
   environment: production
   flow.api.auth0.root: https://api-auth0.akvo.org/flow
   flow.api.root: https://api-auth0.akvo.org/flow
   keycloak.url: https://login.akvo.org/auth
-  lumen.api.authz.probability: "0.1"
+  lumen.api.authz.probability: "1"
```
Comment (dlebrero) [inline k8s/config/prod.yaml]: Are you sure about this change?

Comment (tangrammer) [reply k8s/config/prod.yaml]: this option should be removed after next lumen deployment

Comment (dlebrero) [reply k8s/config/prod.yaml]: But do you want to change it?

**Comment by tangrammer** [comment]
@dlebrero  about your comment "We do not want Keycloak in Test."
but we need keycloak in https://dark-lumencitest.akvotest.org so cypress could hit keycloak

File: k8s/config/prod.yaml (hunk 1 above)
Comment (dlebrero) [reply k8s/config/prod.yaml]: Apologies, now I see your comment from 4 hours ago.

**Comment by dlebrero** [comment]
> @dlebrero about your comment "We do not want Keycloak in Test."
> but we need keycloak in https://dark-lumencitest.akvotest.org so cypress could hit keycloak

Test env should be as close to production as possible, unless a reason of major force.

**Pull request closed without merging**